docs/resources/Channel.md

```diff
@@ -28,7 +28,7 @@ Represents a guild or DM channel within Discord.
 | last_pin_timestamp?                 | ?ISO8601 timestamp                                                          | when the last pinned message was pinned. This may be `null` in events such as `GUILD_CREATE` when a message is not pinned.                                                                    |
 | rtc_region?                         | ?string                                                                     | [voice region](#DOCS_RESOURCES_VOICE/voice-region-object) id for the voice channel, automatic when set to null                                                                                |
 | video_quality_mode?                 | integer                                                                     | the camera [video quality mode](#DOCS_RESOURCES_CHANNEL/channel-object-video-quality-modes) of the voice channel, 1 when not present                                                          |
-| message_count?                      | integer                                                                     | number of messages (not including the initial message or deleted messages) in a thread (if the thread was created before July 1, 2022, it stops counting at 50)                               |
+| message_count?                      | integer                                                                     | number of messages (not including the initial message or deleted messages) in a thread. Threads created before July 1, 2022 may have an inaccurate message count.                                |
```

Comment:
Fine with the info but while it's being edited, could we pull it into an asterisk at the bottom of the table instead?

Reply:
https://github.com/discord/discord-api-docs/pull/5437/commits/0bba269e1718d9d0c3b9ead8e301f45c20e59be6